src/loaders/tiffPyramidLoader.js

```diff
@@ -67,4 +76,26 @@ export default class TiffPyramidLoader {
       (is32Bits && new Uint32Array(tile.data));
     return data;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  onTileError(err) {
+    // Handle geotiff-specific tile errors
+    console.error(err);
+  }
```

Comment:
Here we define how to handle errors for each library. Geotiff allows you to request any positive tile indices, so we don't need to handle any errors for invalid bounds, but want do want to propagate any errors that result for debugging.